app/js/components/ArbitrationList.jsx

```diff
@@ -0,0 +1,68 @@
+/*global web3*/
+import React from 'react';
+import {Card, CardBody, CardHeader, CardTitle, Table, Alert} from 'reactstrap';
+import PropTypes from 'prop-types';
+import ArbitrationResult from "./ArbitrationResult";
+import {ARBITRATION_UNSOLVED} from "../features/arbitration/constants";
+
+function getArbitrationState(escrow) {
+  if(escrow.arbitration.open && escrow.arbitration.result === ARBITRATION_UNSOLVED){
+    return <p className="text-danger">In arbitration</p>;
+  }
+  return <p className="text-success">Arbitration completed</p>;
+}
+
+const Address = (props) => <span title={props.address}>{props.address.substring(0, 6) + "..." + props.address.substring(38)}</span>;
+Address.propTypes = {
+  address: PropTypes.string
+};
+
+const ArbitrationList = (props) => <Card className="mt-2">
```

Comment:
To not break coloration here:
```
const ArbitrationList = (props) => (
  component goes here
);
```
Using parenthesis (same in below components)

Reply:
+1